README.md

```diff
@@ -557,6 +558,66 @@ For detailed documentation on the InstructLab LLMs and their functions, see the
+   export ILAB_FEATURE_SCOPE=DevPreviewNoUpgrade
+   ```
+
+#### Converting documents
+
+Once you have enabled developer preview features, the first step is to convert your content into a structured form and then index that content for RAG to use with `ilab model chat`.  There are three options for conversion:
+
+*Converting documents from a directory:* If you have all the source documents (e.g., PDF, Markdown) that you want to convert in a single local directory, you can convert them:
+
+   ```shell
+      ilab rag convert --input-directory <path-to-document-directory> --output-directory <path-to-converted-document-directory> 
+   ```
+
+This capability has been tested with PDF, Markdown, and Microsoft Word format documents.  Any of the [formats supported by Docling](https://ds4sd.github.io/docling/reference/document_converter/#docling.document_converter.InputFormat) should work.
+
+*Converting documents from a taxonomy:* If you do not specify an input directory, the `convert` command will pull from your taxonomy instead.  For example, to use all of the documents in all of the knowledge nodes in your taxonomy you can run:
+
+   ```shell
+   ilab rag convert --taxonomy-base=empty --output-directory <path-to-converted-document-directory> 
+   ```
+
+*Converting documents within SDG:* The SDG commands will convert documents along with its primary goal (to generate synthetic data).  If you ran any configuration of `ilab data generate`, then the knowledge documents in the taxonomy that generated synthetic data will already be converted to a structured form.  In this case, there is no need to run the `ilab rag convert` command again.  For example, the following command mentioned in the [🚀 Generate a synthetic dataset](#-generate-a-synthetic-dataset) section converts all of the documents referenced in the entire contents of your taxonomy:
+
+   ```shell
+   ilab data convert --taxonomy-base=empty
+   ```
+
+In the last two examples, you can also omit the `--taxonomy-base=empty` flag. In this case, the conversion will only use documents referenced by the new or modified YAML files within your taxonomy rather than all of the documents from the entire taxonomy.
+
+#### Ingesting documents
+
+Once documents are converted, you can use the `ilab rag ingest` command to ingest the converted content into a vector database index file.
+
+*Ingesting documents from SDG:* By default, `ilab rag ingest` pulls the content to ingest from where SDG stores them, so if you converted documents within SDG (see above) you can ingest them using the following command:
+
+   ```shell
+   ilab rag ingest
+   ```
+
+*Ingesting documents from ilab rag convert*: Alternatively, if you ran `ilab rag convert` to convert your documents, without SDG, then the output directory you used for `ilab rag convert` should be the input directory you specify for `ilab rag ingest`, e.g.:
+
+   ```shell
+   ilab rag ingest --input-directory=/my/converted/documents/directory/goes/here/
+   ```
```

Comment:
so might be better to keep the same with above `<path-to-converted-document-directory> `?

Reply:
should be `--input-dir`